components/Autocomplete/index.jsx

```diff
@@ -0,0 +1,260 @@
+import React, {PropTypes} from 'react';
+import Autosuggest from 'react-autosuggest';
+import Scroll from 'react-scroll';
+
+import styles from './index.scss';
+
+const isMobile = window.outerWidth < 550;
+
+export default class Autocomplete extends React.Component {
+  static propTypes = {
+    data: PropTypes.object,
+    disabled: PropTypes.bool,
+    error: PropTypes.string,
+    id: PropTypes.string.isRequired,
+    inputStyle: PropTypes.string,
+    label: PropTypes.string.isRequired,
+    labelHidden: PropTypes.bool,
+    name: PropTypes.string.isRequired,
+    noSuggestionText: PropTypes.string.isRequired,
+    onGetSuggestions: PropTypes.func,
+    placeholder: PropTypes.string,
+    query: PropTypes.object,
+    required: PropTypes.bool,
+    requiredLabel: PropTypes.string,
+    scrollTo: PropTypes.bool,
+    submitOnEnter: PropTypes.bool,
+    suggestionsFooter: PropTypes.any,
+    value: PropTypes.string
+  };
+
+  static defaultProps = {
+    error: null,
+    inputStyle: 'inline',
+    noSuggestionText: 'No results found',
+    placeholder: '',
+    query: {},
+    scrollTo: true
+  }
+
+  state = {
+    noSuggestions: false,
+    showError: false,
+    suggestions: [],
+    value: this.props.value
```

Comment:
If we have this in the state and it is referencing props, then we should maybe default it to empty string?